### PR TITLE
added card component

### PR DESCRIPTION
**Pull request by Swapnilkota108**
Here's a preview of the components :-
https://codesandbox.io/s/hungry-shirley-tx2ms?file=/index.html

### CODE REVIEW

**Comment by preethasri** [comment]
hey swapnil check with the url you have provided for preview.  i think things are not aligned

**Comment by Chakravarthi0** [comment]
Hey Swapnil, there is some issue with the link which you have shared. Can you check it once?

**Comment by Swapnilkota108** [comment]
https://tx2ms.csb.app/

File: components/Card/card.css [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+}
+
+
+:root{
+    --primary-color:#775ada;
+    --error-color:#e12929e9;
+    --success-color:#4DB351;
+    --warning-color:#FF9800;
+    --white-color:#fff;
+    --gray-color:rgba(83, 81, 81, 0.925);
```
Comment (mimansha-swarup) [inline components/Card/card.css]: typo: it should be grey

File: components/Card/card.css [hunk 2]
```diff
@@ -0,0 +1,229 @@
+@import url('https://fonts.googleapis.com/css2?family=Amatic+SC:wght@700&family=Andika&family=Architects+Daughter&family=Balsamiq+Sans&family=Dancing+Script:wght@600&family=Dongle&family=Lato&family=Open+Sans:wght@300&family=Roboto:wght@400;500&display=swap');
```
Comment (mimansha-swarup) [inline components/Card/card.css]: why you are repeating this importing font and defining color.
do it in one separate file and import that file

File: components/Card/card.css [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+    --gray-color:rgba(83, 81, 81, 0.925);
+} 
+
+
+ 
+
+.card{
+    height: max-content;
+    width: 21rem;
+    box-shadow: 2px 2px 2px gray;
```
Comment (mimansha-swarup) [inline components/Card/card.css]: make variable for box shadow too 
it will maintain consistency

File: components/Card/card.css [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+        color: var(--error-color);
+    }
+    .gray{
+       color: var(--gray-color);
+    }
+
+
+/* .card-product{
+    height: max-content;
+     width: max-content;
```
Comment (mimansha-swarup) [inline components/Card/card.css]: remove unnecessary comment

File: components/Card/card.css [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+    box-shadow: 1px 1px 1px var(--gray-color);
+    margin: 1rem;
+}    
+    .card-header{
+        display: flex;
+        width: 100%;
+    }
+        .card-header img{
+            width: 100%;
+        }
```
Comment (mimansha-swarup) [inline components/Card/card.css:109]: suggestion: maintain indentation of code
not gonna affect anything but will increase code readability

**Review by mimansha-swarup — COMMENTED** [review]
few suggestions

**Pull request merged**